cloudflare/schema_cloudflare_devices.go

```diff
@@ -0,0 +1,92 @@
+package cloudflare
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resoureceCloudflareDevicesSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_id": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"devices": {
+			Type:     schema.TypeList,
+			Computed: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"id": {
+						Type:        schema.TypeString,
+						Optional:    true,
+						Description: "Device ID.",
```

Comment:
thanks for doing the `Description` fields, this will make life alot easier once we generate the docs.